models/issue.go

```diff
@@ -1122,7 +1122,7 @@ func GetIssueStats(opts *IssueStatsOptions) *IssueStats {
 	}
 
 	switch opts.FilterMode {
-	case FM_ALL, FM_ASSIGN:
+	case FM_YOUR_REPOSITORIES, FM_ASSIGN:
 		stats.OpenCount, _ = countSession(opts).
 			And("is_closed = ?", false).
 			Count(&Issue{})
```

Comment:
I read up a bit on this distinction, do not fully understand it yet but I will do as you request ;).

From what I understand, &T{} only works for certain types (structs, slices, etc.) whereas new(T) works for any type. Is that the reason you prefer the latter?